tests/integration/components/flash-message-test.js

```diff
@@ -111,4 +111,29 @@ if (parseFloat(Ember.VERSION) > 2.0) {
 
     clock.restore();
   });
+
+  test('a custom component can use the close closure action', function(assert) {
+    assert.expect(3);
+
+    let destroyMessage = sinon.spy();
+    this.set('flash', FlashMessage.create({
+      message: 'flash message content',
+      sticky: true,
+      destroyOnClick: false,
```

Comment:
uh oh right, destroyOnClick=true is the default :/
I was surprised that was true when I first read the docs for this. And I'm surprised again just now haha (I wonder if other users are surprised by this default too?)

Hmm - we could at least document it in the README.md next to an example that uses `close`?